.github/workflows/linux.yml

```diff
@@ -293,6 +295,11 @@ jobs:
             configure: --enable-debug --with-ssl --with-libssh2 --with-libidn2 --with-gssapi --enable-ldap --with-libpsl --disable-threaded-resolver --enable-ares --enable-httpsrr --disable-unity
             container: 'alpine:3.20'
 
+          - name: 'openldap-static'
+            install_steps: openldap-static
+            install_packages: libssl-dev
+            configure: --disable-shared --with-openssl --enable-static --with-ldap=/home/runner/openldap-static --with-ldap-lib=ldap --with-lber-lib=lber
+
```

Comment:
To avoid adding a new CI job, would it work to enable this openldap
option in an existing one? The 'openssl clang krb5' job seems like a
possible candidate, also for `--disable-shared` to make the build
static only.

Also `--enable-static` is the default, and it's safe to drop from the 
configuration.

Reply:
Given that these run in parallel, is there a particular problem in having another matrix option? 

I'm also a big fan of explicit vs implicit, so having --enable-static is more descriptive when looking at the test.

Reply:
Each job is consuming resources. Merging jobs that are possible to merge
saves resources and also save maintenance.